src/pages/Index.vue

```diff
@@ -2,18 +2,120 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    
+    <header class="header">
+    </header>
+      <body>
+        <b-container class="index-container">
+        <b-row>
+          <b-col>
+        <h1>Mentorship...for you!</h1>
 
-    <p>Mentorship...for you!</p>
+        <p>An all in one program targeted to speed up your growth!</p>
+        <a href="https://forms.gle/482EoBYtJozWpYCf9" class="btn" role="button" target="_blank">Apply Now</a>
+            </b-col>
+            <b-col>
+              <img src="/images/illustration.png" alt="two people sitting in front of a computer">
+          </b-col>
+          </b-row>
+        </b-container>
+    
+    <b-container class="index-container">
+     <b-card-group deck>
+         <b-card class="index-card"
+            title="Community"
+           img-src="/images/gcc2019-grouppicture.jpg"
+           img-alt="Community members at Galaxy Community Conference, 2019"
+           img-top
+           border-radius="10px"
```

Comment:
I would remove the borders as nothing else have borders

Reply:
that's true...thanks for catching that!